### DISCUSSION TITLE
After deleting the vector, the acquired set is not correct

### DISCUSSION THREAD

**Discussion by zhenzi0322** [milvus-io/milvus#19183] [Q&A and General discussion]
![image](https://user-images.githubusercontent.com/34839719/189853841-5d3c5910-e081-4b8b-b318-8b6052d35c67.png)

![image](https://user-images.githubusercontent.com/34839719/189853942-bfa6cf46-fec6-407d-87f2-b7da4ae6bd82.png)

Normally there should be 39 vectors in the set. How can I get the number of vectors in the set after deleting them?

**Comment by yhmo** [comment]
The `num_entities` cannot count in the deleted entities for some technical reasons. Sounds a bit funny, but it is really a technical limitation.
We are plan to provide a new API "count()" to calculate a real number of entities, might be available on v2.2.

**Reply by zhenzi0322** [reply]
OK, when is the expected update of version 2.2? Can you send me an email if pymilvus updates version 2.2

**Reply by yhmo** [reply]
I suppose it will be released in Oct, optimistically. I am not sure I can remember at that time.

**Reply by zhenzi0322** [reply]
> I suppose it will be released in Oct, optimistically. I am not sure I can remember at that time.

Ok.

**Reply by yhmo** [reply]
You can subscribe notification from github.
On the page https://github.com/milvus-io/milvus,  click the "watch" dropdown list on the top-rgiht corner, then click "Custom", and choose "Release" checkbox.
